_posts/2017-06-01-how-to-build-dotnet-core-project-with-vs-code.md

```diff
@@ -0,0 +1,375 @@
+  Generating MSBuild file C:\Projects\dotnetcore-vscode\dotnetcorevscode.Web\obj\dotnetcorevscode.Web.csproj.nuget.g.props.
+  Generating MSBuild file C:\Projects\dotnetcore-vscode\dotnetcorevscode.Web\obj\dotnetcorevscode.Web.csproj.nuget.g.targets.
+  Writing lock file to disk. Path: C:\Projects\dotnetcore-vscode\dotnetcorevscode.Web\obj\project.assets.json
+  Restore completed in 2,41 sec for C:\Projects\dotnetcore-vscode\dotnetcorevscode.Web\dotnetcorevscode.Web.csproj.
+
+  NuGet Config files used:
+      C:\Users\joffr\AppData\Roaming\NuGet\NuGet.Config
+      C:\Program Files (x86)\NuGet\Config\Microsoft.VisualStudio.Offline.config
+
+  Feeds used:
+      https://api.nuget.org/v3/index.json
+      C:\Program Files (x86)\Microsoft SDKs\NuGetPackages\
+
+joffr@JOFFREY-LAPTOP MINGW64 /c/Projects/dotnetcore-vscode
+$ dotnet build
+Microsoft (R) Build Engine version 15.1.1012.6693
+Copyright (C) Microsoft Corporation. All rights reserved.
+
+  dotnetcorevscode.Library -> C:\Projects\dotnetcore-vscode\dotnetcorevscode.Library\bin\Debug\netstandard1.4\dotnetcorevscode.Library.dll
+  dotnetcorevscode.Tests -> C:\Projects\dotnetcore-vscode\dotnetcorevscode.Tests\bin\Debug\netcoreapp1.1\dotnetcorevscode.Tests.dll
+  dotnetcorevscode.Web -> C:\Projects\dotnetcore-vscode\dotnetcorevscode.Web\bin\Debug\netcoreapp1.1\dotnetcorevscode.Web.dll
+
+Build succeeded.
+    0 Warning(s)
+    0 Error(s)
+
+Time Elapsed 00:00:05.96
+```
+
+### <a name='Addprojectsreferencesoneachprojects'></a>Add projects references on each projects
+
+On Visual Studio 2017, if you create a new project, you have to add a reference on it by using ``Right click on a project > Add > Reference > ...`` And on our solution, I have to add few references : 
+* dotnetcorevscode.Web needs a reference on dotnetcorevscode.Library
+* dotnetcorevscode.Tests needs a reference on dotnetcorevscore.Tests
+
+To add a reference on a project, you have to ``dotnet add`` command : 
+
+```cli
+joffr@JOFFREY-LAPTOP MINGW64 /c/Projects/dotnetcore-vscode
+$ dotnet add ./dotnetcorevscode.Web/dotnetcorevscode.Web.csproj reference ./dotnetcorevscode.Library/dotnetcorevscode.Library.csproj
+Reference `..\dotnetcorevscode.Library\dotnetcorevscode.Library.csproj` added to the project.
+
+joffr@JOFFREY-LAPTOP MINGW64 /c/Projects/dotnetcore-vscode
+$ dotnet add ./dotnetcorevscode.Tests//dotnetcorevscode.Tests.csproj reference ./dotnetcorevscode.Library/dotnetcorevscode.Library.csproj
+Reference `..\dotnetcorevscode.Library\dotnetcorevscode.Library.csproj` added to the project.
+```
+
+And I can check the existing references on a project : 
+
+```cli
+joffr@JOFFREY-LAPTOP MINGW64 /c/Projects/dotnetcore-vscode
+$ dotnet list ./dotnetcorevscode.Web/dotnetcorevscode.Web.csproj reference
+Project reference(s)
+--------------------
+..\dotnetcorevscode.Library\dotnetcorevscode.Library.csproj
+```
+
+Now the structure of our solution is ready and we'll ``code .`` ! 
```

Comment:
J'aurais mis un "." après ready et formulé la phrase autrement "let's code" ou "We can start coding"